examples/basics/plotting/grid_holed.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
++---+-------+---+
+| 1 |   1   | 2 |
+|---+-------+---+
+| 3 | Empty | 2 |
++---+-------+---+
+| 3 |   4   | 4 |
++---+-------+---+
```

Comment:
This one collapses to be effectively a 2x2. I guess that's expected based on our constraint defaults? I originally expected the solution to be as depicted here, where the `stretch` of the 1x2 and 2x1 cells pushes it to have some empty space in the middle.